packages/test-framework/src/react-testing-library-support/mocks/countries.ts

```diff
@@ -0,0 +1,24 @@
+// intentionally shortened
+const countries = {
```

Comment:
When wirting shipping tests, we definitely need more countries.
Just leave them for now.